.gitignore

```diff
@@ -1,2 +1,3 @@
 nodeapp/node_modules/
-*.log
\ No newline at end of file
+*.log
+node_modules/
```

Comment:
Good idea to add this here. I also looked into installing the modules globally some more. It sounds like others agree with what we saw earlier: you can install them wherever you want, and then it's best to create symbolic links to avoid duplicate installations.

Here's what I was reading specifically:
https://nodejs.org/en/blog/npm/npm-1-0-global-vs-local-installation/

Reply:
Oh and the `npm link express` thing that they suggest works for me, does it work for you?